Lib9c.Abstractions/ILoadIntoMyGarages.cs

```diff
@@ -0,0 +1,20 @@
+#nullable enable
+
+using System.Linq;
+using System.Security.Cryptography;
+using Libplanet;
+using Libplanet.Assets;
+
+namespace Lib9c.Abstractions
+{
+    public interface ILoadIntoMyGarages
+    {
+        IOrderedEnumerable<(Address balanceAddr, FungibleAssetValue value)>? FungibleAssetValues
+        {
+            get;
+        }
+
+        Address? InventoryAddr { get; }
```

Comment:
Good point. I considered using the avatar index for this part. Because it may be more convenient than using the avatar address.
However, if you look at only the action in isolation, the fact that you have to derive and use the inventory address from the avatar address requires additional information.
In this regard, I thought that what the action requires should be the inventory address itself, and it would be better for the side that creates this action to accept the avatar's inventory or address.

Reply:
Hmm :thinking: that makes sense. OK I'll keep this in mind and reflect that to IAP backend service.
Maybe some extra effort will be needed to answer `who's this?` but I understood what's on focus.

Reply:
How about using the `Memo` property like `TransferAsset` action.

Reply:
I applied the `Memo` idea and you can figure out at #1959.